contracts/VoucherHub.sol

```diff
@@ -253,6 +253,37 @@ contract VoucherHub is
         emit VoucherRefunded(msg.sender, amount);
     }
 
+    /**
+     * Drain funds from voucher if it is expired. Transfer all RLC balance
+     * on PoCo from voucher to voucherHub and burn all credits.
+     * @param voucher address of the expired voucher to drain
+     */
+    function drainVoucher(address voucher) external onlyRole(VOUCHER_MANAGER_ROLE) {
+        VoucherHubStorage storage $ = _getVoucherHubStorage();
+        require($._isVoucher[voucher], "VoucherHub: unknown voucher");
+        uint256 amount = balanceOf(voucher);
+        Voucher(voucher).drain(amount);
+        _burn(voucher, amount);
```

Comment:
Minor, burn then drain (local then external)

Reply:
[Swap internal and external operations order](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/25/commits/dc662abc082a9676b72521dc126c3c91330c1b29)